src/scripts/main.js

```diff
@@ -1,29 +1,51 @@
 'use strict';
 
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  const notification = document.createElement('div');
+
+  notification.classList.add('notification', type);
+  notification.style.position = 'absolute';
+  notification.style.left = `${posRight}px`;
+  notification.style.top = `${posTop}px`;
+
+  const titleElement = document.createElement('h2');
+
+  titleElement.classList.add('title');
+  titleElement.textContent = title;
+  notification.appendChild(titleElement);
+
+  const descriptionElement = document.createElement('p');
+
+  descriptionElement.textContent = description;
+  notification.appendChild(descriptionElement);
+
+  document.body.appendChild(notification);
+
+  setTimeout(() => {
+    notification.remove();
```

Comment:
You don't need to completely remove the element, it will fail the tests, you just need to hide it.